src/System.Data.Common/ref/System.Data.Common.cs

```diff
@@ -351,6 +360,9 @@ public abstract partial class DbDataReader : System.Collections.IEnumerable, Sys
         public abstract bool Read();
         public System.Threading.Tasks.Task<bool> ReadAsync() { return default(System.Threading.Tasks.Task<bool>); }
         public virtual System.Threading.Tasks.Task<bool> ReadAsync(System.Threading.CancellationToken cancellationToken) { return default(System.Threading.Tasks.Task<bool>); }
+        System.Data.IDataReader System.Data.IDataRecord.GetData(int ordinal) { return default(System.Data.IDataReader); }
+        public void Close() { }
+        public DataTable GetSchemaTable() { return default(DataTable); }
```

Comment:
The way it is currently, `IDataReader.GetSchemaTable` is implemented implicitly (public).

I'm wondering if this should instead be an explicit (private) implementation of the `IDataReader.GetSchemaTable` method:

```csharp
DataTable System.Data.IDataReader.GetSchemaTable() { return default(DataTable); }
```

https://msdn.microsoft.com/en-us/library/aa664591(v=vs.71).aspx